src/Plutus/Test/Model/Fork/TxExtra.hs

```diff
@@ -40,14 +41,15 @@ toExtra = Tx mempty
 data Extra = Extra
   { extra'withdraws      :: [Withdraw]
   , extra'certificates   :: [Certificate]
+  , extra'descr          :: Last String
```

Comment:
Let's keep `Tx` for things that relate to Cardano. This is test concept to improve readability. We can use `BchNames` for that or add this info to something that is not related to `Tx`. For example to Stats. But `BchNames` is preferable. It's the place to keep human readably synonyms for pretty printing.